staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -94,7 +116,14 @@ func NewCommandStartWardleServer(ctx context.Context, defaults *WardleServerOpti
 
 	flags := cmd.Flags()
 	o.RecommendedOptions.AddFlags(flags)
-	utilfeature.DefaultMutableFeatureGate.AddFlag(flags)
+
+	wardleEffectiveVersion := utilversion.NewEffectiveVersion("1.2")
+	utilruntime.Must(utilversion.DefaultComponentGlobalsRegistry.Register(apiserver.WardleComponentName, wardleEffectiveVersion, nil, false))
+	_, featureGate := utilversion.DefaultComponentGlobalsRegistry.ComponentGlobalsOrRegister(
+		utilversion.ComponentGenericAPIServer, utilversion.DefaultKubeEffectiveVersion(), utilfeature.DefaultMutableFeatureGate)
+
+	wardleEffectiveVersion.AddFlags(flags, "wardle-")
```

Comment:
To remind us.  this is an example right now and we want to wire an example with wardle specific gates.

`--wardle-emulation-version=1.2 --wardle-features` ?

Reply:
`--feature=wardle:feature-gate=true` where "wardle" is optional as long as there are no conflicts.  any conflict produce a failure and requires specification.

Benefits
1. this keeps all help for all features together
2. it allows to ensure consistent featuregate handling if david and tim refactro the world.
3. allows similar consistency with env var retrieval and (potential) apiserver reading

Reply:
Cobra has a way to support both `--emulation-version=kube=1.31,wardle=1.2` and `--emulation-version=kube=1.31 --emulation-version=wardle=1.2`.  I think we all prefer the former, but I think the library is both.